makefile

```diff
@@ -69,15 +78,16 @@ CPPFLAGS = $(MFEM_CPPFLAGS)
 CXXFLAGS = $(MFEM_CXXFLAGS)
 
 # MFEM config does not define C compiler
-CC     = gcc
-CFLAGS = -O3
+CC     ?= gcc
+CFLAGS ?= -O3
 
-# Optional link flags
-LDFLAGS =
+# Optional compile/link flags
+GLVIS_OPTS ?=
+GLVIS_LDFLAGS ?=
```

Comment:
These two lines also seems unnecessary. Did you add these just to emphasize that these are configurable variables?